### PR TITLE
VALIDATE SGC

### PR DESCRIPTION
**Pull request by solanagemcoin**
# Validate SGC
Name: SOLANA GEM COIN
Symbol: SGC
Mint Address: 2LiQdNty6M68C8LqA4XBQF75W6seB8KSRxq2jAaYjbHW
Decimals: 2
LogoURI: https://arweave.net/aIsws2O-pdDLWCeSfawZrJr3AiRstpVeVi-_e7u4AHg
Community Validated: true

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/SolGemClub/status/1690137732171862016?s=20
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by Timoon21** [comment]
You need to add your token at the end of _validated-tokens.csv_ file. Not on a new file.
Please update your PR (or close it and open another one).

**Comment by Timoon21** [comment]
Btw, some data doesn't matches on-chain data:

Address: OK
Name: NOK ------>  **SOLANA GEM COIN**
Symbol: OK
Decimals: OK
LogoURI: NOK ------>  **https://arweave.net/BIRnbgEd2SmbEd_1c80BWr2DoGba81-CJp5aqJPL7P0**

**Comment by Timoon21** [comment]
> > You need to add your token at the end of _validated-tokens.csv_ file. Not on a new file. Please update your PR (or close it and open another one).
> 
> necessary changes updated

No.
You need to add your token at the end of _validated-tokens.csv_ file. Not in a new file.

**Comment by solanagemcoin** [comment]
> > > You need to add your token at the end of _validated-tokens.csv_ file. Not on a new file. Please update your PR (or close it and open another one).
> > 
> > 
> > necessary changes updated
> 
> No. You need to add your token at the end of _validated-tokens.csv_ file. Not in a new file.

sir i have added to the file can you check it

**Comment by Timoon21** [comment]
> > > > You need to add your token at the end of _validated-tokens.csv_ file. Not on a new file. Please update your PR (or close it and open another one).
> > > 
> > > 
> > > necessary changes updated
> > 
> > 
> > No. You need to add your token at the end of _validated-tokens.csv_ file. Not in a new file.
> 
> sir i have added to the file can you check it

Not the right file. You have to change validated-token.csv file. Don't add new file. Check other Pull request.

**Review by solanagemcoin — COMMENTED** [review]
change updated

**Comment by Timoon21** [comment]
> change updated

Not the right file. You have to change validated-token.csv file. Don't add new file. Check other Pull request.

**Comment by solanagemcoin** [comment]
> > change updated
> 
> Not the right file. You have to change validated-token.csv file. Don't add new file. Check other Pull request.

https://github.com/jup-ag/token-list/pull/283/commits/f49b1e888520378118e0ece09222f053e6506704 update now.

**Comment by Timoon21** [comment]
> > > change updated
> > 
> > 
> > Not the right file. You have to change validated-token.csv file. Don't add new file. Check other Pull request.
> 
> [f49b1e8](https://github.com/jup-ag/token-list/commit/f49b1e888520378118e0ece09222f053e6506704) update now.

Yep good for validated-tokens.csv

But need to remove the other file (Solana gem coin file).

**Comment by solanagemcoin** [comment]
> > > > change updated
> > > 
> > > 
> > > Not the right file. You have to change validated-token.csv file. Don't add new file. Check other Pull request.
> > 
> > 
> > [f49b1e8](https://github.com/jup-ag/token-list/commit/f49b1e888520378118e0ece09222f053e6506704) update now.
> 
> Yep good for validated-tokens.csv
> 
> But need to remove the other file (Solana gem coin file).

sir done

File: validated-tokens.csv [hunk 1]
```diff
@@ -756,3 +756,4 @@ GEM,GEM,E7BGDtpNXUTqPNbZxKHiLowgLddiAeuKcByD7tSnfYWD,5,https://bafkreifpm7rbwohp
 SOUL,SOUL,F6weWmuc1vwdL4u38Ro9jKXHEMjP9BoNdWMF5o5TvtJf,9,https://arweave.net/ipj9IdEAQAXt0LS6lOt7jQY7W4iazsVFBkN60U7M4iQ,true
 Venture Coin,VC,AebrVZPfSH7KPAxPwnuqTZB9QNepdktk7HSSY4SNj7BM,9,https://arweave.net/6HvW8RLbLhBbw09QDRpPd6HX-N3Sk9kF2vvdaZGHExU,true
 NINJA TURTLES,NINJA,DFrJxDoLMYt6bNYeNe8Wrjzj2UPUSLZLEMMYBLuTKcTk,8,https://arweave.net/2J8mJ6N6GCg7BWTHtcPZMEZsF2iuTYBRI9jdtnHS77o,true
+SOLANA GEM COIN, SGC,2LiQdNty6M68C8LqA4XBQF75W6seB8KSRxq2jAaYjbHW,2,https://arweave.net/aIsws2O-pdDLWCeSfawZrJr3AiRstpVeVi-_e7u4AHg,true
```
Comment (Timoon21) [inline validated-tokens.csv:759]:
```suggestion
SOLANA GEM COIN,SGC,2LiQdNty6M68C8LqA4XBQF75W6seB8KSRxq2jAaYjbHW,2,hhttps://arweave.net/1r8SY0_SHxYGUK4WsLZ3SZ0XyGTkt9BzsfEuzDued8k,true
```

**Comment by Timoon21** [comment]
I don't see much support from the community on Twitter. Can you try to get more support on twitter (like, retweet) or on github (comment) ?

**Comment by meijiao** [comment]
Closing -- please see timoon's last comment and reopen the PR when you're ready

**Comment by solanagemcoin** [comment]
You are constantly blocking to disapprove. I guess you are doing this for money.There are many projects like projects and you approve all of them. We have provided all the necessary conditions for listing.

**Pull request closed without merging**